Dockerfile

```diff
@@ -1,22 +1,56 @@
-# This is our base community provided image,
-# with some minimal distribution of linux
-# that comes with NodeJs v10 (and npm) already installed
-FROM node:lts
+###################
+# BUILD FOR LOCAL DEVELOPMENT
+###################
+FROM node:20-alpine As development
 
-# Use this directory _inside_ your Docker container
-# as the base directory of all the next actions
+# Create app directory
 WORKDIR /usr/src/app
 
-# Copy your app's code from the host (your computer) to the container.
-# This assumes a typical setup in which the Dockerfile
-# is in the root of your project
-COPY package.json .
-
-# Install dependencies
-RUN npm install
-COPY webpack ./webpack 
-COPY .stylelintrc .
-COPY .prettierrc .
-COPY tsconfig.json .
-COPY tslint.json . 
-COPY config ./config
\ No newline at end of file
+# Copy application dependency manifests to the container image.
+# A wildcard is used to ensure copying both package.json AND package-lock.json (when available).
+# Copying this first prevents re-running npm install on every code change.
+COPY --chown=node:node package*.json ./
+
+# Install app dependencies using the `npm ci` command instead of `npm install`
+RUN npm ci
+
+# Bundle app source
+COPY --chown=node:node . .
```

Comment:
When setting up I see the  node_modules belonging to the root user still, does this work for you?

Reply:
I'll merge this PR now, but this is definitely something that needs review. Thus, I decided to create an issue: https://github.com/etsiiull/SIMDE/issues/68